highs/interfaces/highs_c_api.cpp

```diff
@@ -1309,32 +1309,32 @@ HighsInt Highs_getIis(void* highs, HighsInt* iis_num_col, HighsInt* iis_num_row,
   *iis_num_col = iis.col_index_.size();
   *iis_num_row = iis.row_index_.size();
   if (col_index != nullptr) {
-    for (size_t i = 0; i < *iis_num_col; i++) {
+    for (size_t i = 0; i < (size_t)*iis_num_col; i++) {
       col_index[i] = iis.col_index_[i];
     }
   }
   if (row_index != nullptr) {
-    for (size_t i = 0; i < *iis_num_row; i++) {
+    for (size_t i = 0; i < (size_t)*iis_num_row; i++) {
       row_index[i] = iis.row_index_[i];
     }
   }
   if (col_bound != nullptr) {
-    for (size_t i = 0; i < *iis_num_col; i++) {
+    for (size_t i = 0; i < (size_t)*iis_num_col; i++) {
       col_bound[i] = iis.col_bound_[i];
     }
   }
   if (row_bound != nullptr) {
-    for (size_t i = 0; i < *iis_num_row; i++) {
+    for (size_t i = 0; i < (size_t)*iis_num_row; i++) {
       row_bound[i] = iis.row_bound_[i];
     }
   }
   if (col_status != nullptr) {
-    for (size_t i = 0; i < ((Highs*)highs)->getLp().num_col_; i++) {
```

Comment:
Isn't this also going to flag up a warning (with some architectures/compilers) since `getLp().num_col_` is `HighsInt`, not `size_t`?